Source/Core/DolphinQt/TAS/TASInputWindow.cpp

```diff
@@ -151,7 +216,6 @@ QSpinBox* TASInputWindow::CreateSliderValuePair(QBoxLayout* layout, int default_
   slider->setRange(0, max);
   slider->setValue(default_);
   slider->setFocusPolicy(Qt::ClickFocus);
-  slider->setInvertedAppearance(invert);
```

Comment:
Why was this (and the `invert` parameter in general) removed? The IR widget used them. (However, it seems like things still behave correctly - the sliders for the IR widget still match the position and aren't reversed, at least on my machine, so I'm not sure what it does; [the docs for `setInvertedAppearance`](https://doc.qt.io/qt-6/qabstractslider.html#invertedAppearance-prop) don't really clarify either.)

Reply:
Previously, the IR sliders were more or less directly mapped to where on the IR camera the sensor bar lights are visible. When you move a Wii Remote right, the lights appear to move to the left from the perspective of the camera – which means that when you move the slider right, we should invert the input so that the light points will actually move to the left. With the new input override API, the sliders are mapped to the Point action, and WiimoteEmu takes care of translating from "location you're pointing at" to "light locations".